src/app/inventory/header/header.component.scss

```diff
@@ -14,7 +14,10 @@
   .cancel {
     float: left;
   }
-  .search{
+  .clean {
+    float: left;
```

Comment:
i don't like floats but i can see that it's deep in the code base, so i will let it go, maybe another time